src/common/Pagination/index.js

```diff
@@ -0,0 +1,35 @@
+import {
+  PageCounter,
+  ButtonText,
+  PrevArrowIcon,
+  Wrapper,
+  PageNumbers,
+  Button,
+  NextArrowIcon,
+} from "./styled";
+
+export const Pagination = () => {
+  return (
+    <Wrapper>
+      <Button>
+        <PrevArrowIcon />
+        <ButtonText>First</ButtonText>
+      </Button>
+      <Button>
+        <PrevArrowIcon />
+        <ButtonText>Previous</ButtonText>
+      </Button>
+      <PageCounter>
+        Page <PageNumbers>1</PageNumbers> of <PageNumbers>500</PageNumbers>
```

Comment:
Word "Page" and "of" should be more "grey" on active buttons, rather than default black